frontend/src/component/layout/MainLayout/NavigationSidebar/NavigationList.tsx

```diff
@@ -33,15 +33,15 @@ const StyledBadgeContainer = styled('div')(({ theme }) => ({
     display: 'flex',
 }));
 
-const EnterprisePlanBadge = () => (
+export const EnterprisePlanBadge = () => (
```

Comment:
Not sure if this should be exported from here for it's usage in AdminHome as it kind of breaks the idea of having components that are self contained in a folder. But maybe double check this with someone with more context than me. This is the ADR I'm referring to https://docs.getunleash.io/contributing/ADRs/front-end/preferred-folder-structure

Reply:
Was planning on adding some context to this. My other option was to copy it. But I chose to export it and add a reference to it because it's being looked at for changing by UX, so referencing it means both places get updated more easily, and I don't add another place to change it

@kwasniew thoughts on this?

Reply:
Can we move it to a separate component that both NavigationList and AdminNavigationItems reference?